internal/flags/flags_test.go

```diff
@@ -19,21 +19,38 @@ package flags
 import (
 	"os"
 	"os/user"
+	"runtime"
 	"testing"
 )
 
 func TestPathExpansion(t *testing.T) {
 	user, _ := user.Current()
-	tests := map[string]string{
-		"/home/someuser/tmp":              "/home/someuser/tmp",
-		"~/tmp":                           user.HomeDir + "/tmp",
-		"~thisOtherUser/b/":               "~thisOtherUser/b",
-		"$DDDXXX/a/b":                     "/tmp/a/b",
-		"/a/b/":                           "/a/b",
-		"C:\\Documents\\Newsletters\\":    "C:\\Documents\\Newsletters\\",
-		"C:\\":                            "C:\\",
-		"\\\\.\\pipe\\\\pipe\\geth621383": "\\\\.\\pipe\\\\pipe\\geth621383",
+	var tests map[string]string
+
+	if runtime.GOOS == "windows" {
+		tests = map[string]string{
+			"/home/someuser/tmp":              "\\home\\someuser\\tmp",
+			"~/tmp":                           user.HomeDir + "\\tmp",
+			"~thisOtherUser/b/":               "~thisOtherUser\\b",
+			"$DDDXXX/a/b":                     "\\tmp\\a\\b",
+			"/a/b/":                           "\\a\\b",
+			"C:\\Documents\\Newsletters\\":    "C:\\Documents\\Newsletters\\",
+			"C:\\":                            "C:\\",
+			"\\\\.\\pipe\\\\pipe\\geth621383": "\\\\.\\pipe\\\\pipe\\geth621383",
```

Comment:
This (by which I mean all of them) would be better expressed with backticked strings, IMO
```suggestion
			`\\.\pipe\\pipe\geth621383`: `\\.\pipe\\pipe\geth621383`,
```